applications/GeoMechanicsApplication/custom_utilities/transport_equation_utilities.hpp

```diff
@@ -49,6 +49,25 @@ class GeoTransportEquationUtilities
                RelativePermeability * PermeabilityUpdateFactor * IntegrationCoefficient;
     }
 
+    template <unsigned int TDim, unsigned int TNumNodes>
+    static inline BoundedMatrix<double, TNumNodes * TDim, TNumNodes> CalculateCouplingMatrix(
+        const Matrix& rB, const Vector& rVoigtVector, const Vector& rNp, double BiotCoefficient, double BishopCoefficient, double IntegrationCoefficient)
+    {
+        return CalculateCouplingMatrix(rB, rVoigtVector, rNp, BiotCoefficient, BishopCoefficient,
+                                       IntegrationCoefficient);
+    }
+
+    static inline Matrix CalculateCouplingMatrix(const Matrix& rB,
+                                                 const Vector& rVoigtVector,
```

Comment:
VoigtVector will never be different, therefore it may be skipped from the argument list.

Reply:
I suggest to create an issue to remove a calculation of VoightVector again and again.